modules/brightMountainMediaBidAdapter.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+```
+Module Name:  Bright Mountain Media Bidder Adapter
+Module Type:  Bidder Adapter
+Maintainer: example@bmtm.org
+```
+
+# Description
+
+Connects to Bright Mountain Media exchange for bids.
+
+Bright Mountain Media bid adapter currently supports Banner.
+
+# Test Parameters
+```
+ var adUnits = [
+      code: 'placementid_0',
+      mediaTypes: {
+          banner: {
+              sizes: [[300, 250]]
+          }
+      },
+      bids: [
+          {
+              bidder: 'brightmountainmedia',
+              params: {
+                  placement_id: 5f21784949be82079d08c,
```

Comment:
to be a valid JSON, the placement_id value should be wrapped in quotes: 

```
placement_id: '5f21784949be82079d08c',
```

Reply:
Fixed in 1e9b7dcdd43bf215fada66f1287c4404e03f5032